mac/KeymanEngine4Mac/KeymanEngine4Mac/KME/OnScreenKeyboard/OSKView.m

```diff
@@ -359,7 +359,14 @@ - (void)resizeOSKLayout {
 
 - (void)keyAction:(id)sender {
     KeyView *keyView = (KeyView *)sender;
-    NSUInteger keyCode = [keyView.key keyCode];
+    NSUInteger keyCode;
+    @try {
+        keyCode = [keyView.key keyCode];
+    }
+    @catch (NSException *exception) {
+        NSLog(@"Exception in keyAction:sender - %@", exception);
```

Comment:
It's certainly a hammer. Not sure if we'll be able to identify the actual problem once this is in place? Certainly only if we capture log files... so am uncertain as to how appropriate this approach is.